docs/source/tutorial/representations.rst

```diff
@@ -69,24 +69,31 @@ predictions for entities not seen during training.
 
     from pykeen.pipeline import pipeline
     from pykeen.datasets import get_dataset
-    from pykeen.nn.representation import EmbeddingSpecification, LabelBasedTransformerRepresentation
+    from pykeen.nn.init import LabelBasedInitializer
+    from pykeen.nn import Embedding
     from pykeen.models import ERModel
 
     dataset = get_dataset(dataset="nations")
-    entity_representations = LabelBasedTransformerRepresentation.from_triples_factory(
-        triples_factory=dataset.training,
+
+    label_initializer = LabelBasedInitializer.from_triples_factory(dataset.training)
```

Comment:
@mberr can you check if this works properly?

Perhaps we should make some helper methods for these class-based initializer functions for generating Embeddings that aren't trainable since we have re-written this code several times

Reply:
https://github.com/pykeen/pykeen/pull/979/commits/2c48f0d4e2c015f1700c9ca92851d088481b1b12

Reply:
proper work: https://github.com/pykeen/pykeen/pull/979/commits/2e1ad7929ed96528d08a9488d4346c79e3e89596